gensim/scripts/word2vec2tensor.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2016 Loreto Parisi <loretoparisi@gmail.com>
+# Copyright (C) 2016 Silvio Ogliastri <silvio.olivastri@gmail.com>
+# Licensed under the GNU LGPL v2.1 - http://www.gnu.org/licenses/lgpl.html
+
+"""
+USAGE: $ python -m gensim.scripts.word2vec2tensor --input <Word2Vec model file> --output <TSV tensor filename prefix>
+Where:
+    <Word2Vec model file>: Input Word2Vec model
+    <TSV tensor filename prefix>: 2D tensor TSV output file name prefix
+Output:
+    The script will create two TSV files. A 2d tensor format file, and a Word Embedding metadata file. Both files will
+    us the --output file name as prefix
+This script is used to convert the word2vec format to Tensorflow 2D tensor and metadata formats for Embedding Visualization
+For more information about TensorBoard format see: https://www.tensorflow.org/versions/master/how_tos/embedding_viz/
```

Comment:
Please add instructions on how to see the viz in tensorboard, i.e. "Launch `tensorboard --logdir=dir_with_tsv"